plenum/server/catchup/cons_proof_service.py

```diff
@@ -49,6 +49,8 @@ def __init__(self,
         self._already_asked_for_cons_proofs_without_timeout = False
         self._requested_consistency_proof = set()
         self._last_txn_3PC_key = {}
+        self._ledger_status_timer = None
```

Comment:
I think, that more cleanly, if we will initiate RepeatingTimer there with `active=False`, and then just call `some_timer.start()` or `some_timer.stop()`

Reply:
I'm not sure, because for every case we initiate a new RepeatingTimer  because RepeatingTimer._interval can be changed. But at this moment we have no method for this. Should we change RepeatingTimer?

Reply:
ok, i think, that initiating timer on __init__ state it's not a good idea, because interval for repeating can be changed.